filter_noise_messages.py

```diff
@@ -0,0 +1,35 @@
+import json
+import argparse
+
+from core_data_modules.cleaners import somali
+from core_data_modules.logging import Logger
+from core_data_modules.util import IOUtils
+
+log = Logger(__name__)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Filter out noise messages from Coda Files",
+                                     # Support \n and long lines
+                                     formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument("coded_dir_path", metavar="coded-dir-path",
+                        help="Directory to read then write coded Coda files to")
+
+    args = parser.parse_args()
+    coded_dir_path = args.coded_dir_path
+
+    with open(coded_dir_path) as f:
+        messages = json.load(f)
+
+    not_noise = []
+    def filter_noise_message(messages):
```

Comment:
Defining a function in the __name__ == "__main__" case with the same name as the file then calling it only once feels a bit redundant. I don't think you need to define a function, just outdent all the code below

Reply:
updated thanks